### PR TITLE
allow to optimize jpeg files

### PR DESCRIPTION
**Pull request by buddhaCode**
Hey Froshis,
I really like your TinyPNG integration. Mostly because it's not so invasive like the one in the community store. Happy to see that you already fixed the cached config reader, because the latest release of the plugin doesn't work at all.

Besides you're recommending another service to optimize jpeg images, there might be some ppl out there, who like to use TinyPng for this. Like me :)

This PR adds a configuration to allow jpeg optimization, too.

### CODE REVIEW

File: Resources/config.xml [hunk 1]
```diff
@@ -42,5 +42,24 @@
...
+            <label>Use for image formats</label>
+            <label lang="de">Nutze für Bildformate</label>
+            <value>image/png</value>
+            <store>
+                <option>
+                    <value>image/png</value>
+                    <label>PNG</label>
+                </option>
+                <option>
+                    <value>image/jpeg</value>
```
Comment (tinect) [inline Resources/config.xml:56]: Thanks for your work! This should result for savety in image/jpg, too.

File: Resources/services.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
         <service id="frosh_tinypng_optimizer.config" class="Shopware\Components\Plugin\CachedConfigReader">
             <factory service="shopware.plugin.cached_config_reader" method="getByPluginName"/>
-            <argument type="string">FroshTinyPngMediaOptimizer</argument>
+            <argument type="string">%frosh_tiny_png_media_optimizer.plugin_name%</argument>
```
Comment (tinect) [inline Resources/services.xml:22]: nothing said :-)

**Review by tinect — CHANGES_REQUESTED** [review]

File: Resources/config.xml (hunk 1 above)
Comment (buddhaCode) [reply Resources/config.xml:56]: Nope. Won't fix. Haha. Ever wanted to say that.
There is no mime type image/jpg. Basicly, PHP is using parts of the `file` command to guess the mime type which is based in the following src files:
https://git.in-ulm.de/cbiedl/file/src/master/magic/Magdir/jpeg#L13
For uploading file in rare cases the Symfony HTTP Foundation component is used, which is based on an apache source:
http://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types

File: Resources/services.xml (hunk 2 above)
Comment (buddhaCode) [reply Resources/services.xml:22]: Hehe. You defined 5.2.17 as a minimum ;)

File: Resources/config.xml (hunk 1 above)
Comment (tinect) [reply Resources/config.xml:56]: Passed all tests :trollface: 
Thank you!

**Review by tinect — APPROVED** [review]

**Pull request merged**